### DISCUSSION TITLE
MVT styling

### DISCUSSION THREAD

**Discussion by imakihi** [TerriaJS/terriajs#5542] [Q&A] [answered]
Hi there.
Is there any way to apply styling to MVT layer? I could show MVT polygons with fill and border colors. What I want to do is to use data attributes to color polygons.

Thank you!
Hiroo

**Comment by imakihi** [comment, marked as answer]
FYI.
We ended up using Composite catalog item to style our layers.  We selected on attribute and style it. The we repeated it and composited them.
Hiroo

**Reply by AnaBelgun** [reply]
hi @imakihi we're working on this https://github.com/TerriaJS/terriajs/issues/5600 as part of v8 release. Does that help?

**Reply by imakihi** [reply]
Hi @AnaBelgun . I just went to the V8 documentation and saw new Mapbox related catalog items. I need to spend time to go through new items. 
I really look for the V8 release you guys working on!
Best regards.

Hiroo
